lang/en/admin.php

```diff
@@ -793,8 +793,7 @@
 $string['manageformats'] = 'Manage course formats';
 $string['manageformatsgotosettings'] = 'Default format can be changed in {$a}';
 $string['managelang'] = 'Manage';
-$string['questionbanks'] = 'Question bank plugins';
-$string['manageqbanks'] = 'Manage question bank plugins';
+$string['manageqbanks'] = 'Manage question banks';
```

Comment:
As per discussed in the catch-up, we don't want to edit this file in this PR, so please leave as it is now.

Reply:
Rolled back